### PR TITLE
Fix#273

### LINKED ISSUE #273: Jump on values where the ! operator returns true in gdscript fails
Jumping on values such as 0, where in gdscript the `!`-operator makes them return `true` fails in execution with a cryptic error message. This is because of line 154 in Executed.gd, it should probably say `if var_ == null` instead. Really `get_variable` should return a bool value saying if it found anything, or something similar to that.

**To Reproduce**
```
char.var = 0
jump test if char.var == 0

test:
    char "This won't ever happen"
```

**Expected behavior**
You should be able to jump on zero values

**Runtime:**
 - Godot version 4.1.3
 - Rakugo Version 2.2

### PR DESCRIPTION
**Pull request by Jeremi360**
- Fixes #273
- Fixes/add some comments
- Removes Rakugo.tscn as we don't use it any more in 2.x

### CODE REVIEW

File: Test/TestExecuter/fix_273/Fix273.gd
Comment (theludovyc) [inline Test/TestExecuter/fix_273/Fix273.gd:1]: Hello,

This is not a GUT test

**Review by theludovyc — CHANGES_REQUESTED** [review]

File: addons/Rakugo/Rakugo.gd
Comment (Mojken) [inline addons/Rakugo/Rakugo.gd:1]: This is a very big diff for a very small change in logic. Consider separating the codestyle changes into a separate PR.

EDIT: I'm second-guessing this upon reading the RP comment again. Seems this is your release flow?

File: Test/TestExecuter/fix_273/fix273.rk
Comment (Mojken) [inline Test/TestExecuter/fix_273/fix273.rk:1]: Consider simplifying this test further by removing the character, making var be a global variable instead
```
var = 0
jump test if var == 0

test:
	"This won't ever happen"
```

File: addons/Rakugo/Rakugo.gd
Comment (Jeremi360) [reply addons/Rakugo/Rakugo.gd:1]: My bad, I'm leazy, and want to fix bugs asap,
so often when I have other small changes (like comments, removing some old stuff),
I don't bother myself with spliting it into separed PRs.
I know I should.

File: Test/TestExecuter/fix_273/fix273.rk
Comment (Jeremi360) [reply Test/TestExecuter/fix_273/fix273.rk:1]: You are right, I added character to be sure it wasn't some character raltead bug.
Now as I'm making this test in to proper GUT(Godot Unit Test), I will simplelfy it.

File: Test/TestExecuter/fix_273/Fix273.gd
Comment (Jeremi360) [reply Test/TestExecuter/fix_273/Fix273.gd:1]: I made it into GUT (last commit), but GUT framework is broken,
as it crashes due some, bad code with singnals:
 
image

Comment (Jeremi360) [reply Test/TestExecuter/fix_273/Fix273.gd:1]: I made it into GUT script, but GUT can't find/run this script :/
I don't know way as it is in `Test/TestExecuter/` which is in our GUT confing.

image

Comment (Jeremi360) [reply Test/TestExecuter/fix_273/Fix273.gd:1]: Now it is and it is passed:
image

**Review by theludovyc — APPROVED** [review]

**Comment by theludovyc** [comment]
Hello @Mojken ,

Welcome and thank you for the review :blush:

**Pull request merged**